superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the NV Bandwidth Test."""
+
+import os
+import re
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class NvBandwidthBenchmark(MicroBenchmarkWithInvoke):
+    """The NV Bandwidth Test benchmark class."""
+
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'nvbandwidth'
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
```

Comment:
does this test have no arguments at all?

Reply:
Yes.
If n arguments are given, nvbandwidth will run all the benchmarks by default.

Reply:
Just reviewed the documentation for nvbandwidth. We could consider adding options to modify some default parameters, such as 'copy buffer size' and 'iteration rounds'...

Reply:
The buffer size is definitely important.